cli/azd/resources/alpha_features.yaml

```
@@ -12,3 +12,5 @@
   description: "Extends provisioning providers with azd operations."
 - id: aca.persistIngressSessionAffinity
   description: "Do not change Ingress Session Affinity when deploying Azure Container Apps."
+- id: deployment.stacks
+  description: "Enables Azure Deployment stacks for all ARM/Bicep based deployments"
```

suggest lowercase on deployment stacks to match [docs](https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/deployment-stacks?tabs=azure-powershell).

```suggestion
  description: "Enable Azure deployment stacks for ARM/Bicep-based deployments."
```